crates/rust-analyzer/src/cli/scip.rs

```diff
@@ -599,4 +598,22 @@ pub mod example_mod {
             "rust-analyzer cargo main . MyTypeAlias#",
         );
     }
+
+    #[test]
+    fn documentation_matches_doc_comment() {
+        let s = "/// foo\nfn bar() {}";
+
+        let mut host = AnalysisHost::default();
+        let change_fixture = ChangeFixture::parse(s);
+        host.raw_database_mut().apply_change(change_fixture.change);
+
+        let analysis = host.analysis();
+        let si = StaticIndex::compute(&analysis);
+
+        let file = si.files.first().unwrap();
+        let (_, token_id) = file.tokens.first().unwrap();
+        let token = si.tokens.get(token_id.clone()).unwrap();
```

Comment:
```suggestion
        let token = si.tokens.get(token_id.unwrap());
```

Reply:
That doesn't compile:

```
error[E0599]: no method named `unwrap` found for reference `&ide::TokenId` in the current scope
   --> crates/rust-analyzer/src/cli/scip.rs:615:44
    |
615 |         let token = si.tokens.get(token_id.unwrap());
    |                                            ^^^^^^ method not found in `&TokenId`
```

I've changed the PR to dereference the TokenId instead, which should hopefully also make clippy happy.